records_mover/db/driver.py

```diff
@@ -42,43 +65,63 @@ def schema_sql(self,
         """
         # http://docs.sqlalchemy.org/en/latest/core/reflection.html
         table_obj = self.table(schema, table)
-        return str(CreateTable(table_obj, bind=self.db))
+        return str(CreateTable(table_obj))
 
     def varchar_length_is_in_chars(self) -> bool:
         """True if the 'n' in VARCHAR(n) is represented in natural language
         characters, rather than in post-encoding bytes.  This varies by database -
         override it to control"""
         return False
 
-    def set_grant_permissions_for_groups(self, schema_name: str, table: str,
+    def set_grant_permissions_for_groups(self,
```

Comment:
we probably shouldnt change this but its really weird to me that this method takse a `db` as input, even though the constructor of the dbdriver already does this.

Reply:
is there a reason for this? why can't we just use the existing db stuff in `self`? is there a use case where we need a different db on the same driver object?

Reply:
I can't think of a reason that it would take a different db, unless for some reason this db_driver needed to set permissions for a db it was writing to, but I don't see why that wouldn't be done by that db's db_driver. I agree this is super bizarre, I also agree that it'd be maybe more of a pain than it's worth to change.